priv/repo/migrations/20231115204622_add_user_likes_table.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Finsta.Repo.Migrations.AddUserLikesTable do
+  use Ecto.Migration
+
+  def change do
+    create table(:likes) do
+      add :user_id, references(:users, on_delete: :nothing)
+      add :post_id, references(:posts, on_delete: :delete_all)
```

Comment:
I changed this to be `on_delete: :delete_all` to prevent an error when trying to delete a post with likes due to the foreign key constraint